
-------------------------------header----------------------
2019-01-30 09:23:50,043 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"CapitalAccount":"31685803002369318"},result:System.InvalidOperationException: 数组的反序列化不支持类型“DaZhongTransitionLiquidation.Common.BankFlowData”。
   在 System.Web.Script.Serialization.ObjectConverter.ConvertListToObject(IList list, Type type, JavaScriptSerializer serializer, Boolean throwOnError, IList& convertedList)
   在 System.Web.Script.Serialization.ObjectConverter.ConvertObjectToTypeInternal(Object o, Type type, JavaScriptSerializer serializer, Boolean throwOnError, Object& convertedObject)
   在 System.Web.Script.Serialization.ObjectConverter.ConvertObjectToTypeMain(Object o, Type type, JavaScriptSerializer serializer, Boolean throwOnError, Object& convertedObject)
   在 System.Web.Script.Serialization.ObjectConverter.AssignToPropertyOrField(Object propertyValue, Object o, String memberName, JavaScriptSerializer serializer, Boolean throwOnError)
   在 System.Web.Script.Serialization.ObjectConverter.ConvertDictionaryToObject(IDictionary`2 dictionary, Type type, JavaScriptSerializer serializer, Boolean throwOnError, Object& convertedObject)
   在 System.Web.Script.Serialization.ObjectConverter.ConvertObjectToTypeInternal(Object o, Type type, JavaScriptSerializer serializer, Boolean throwOnError, Object& convertedObject)
   在 System.Web.Script.Serialization.ObjectConverter.ConvertObjectToTypeMain(Object o, Type type, JavaScriptSerializer serializer, Boolean throwOnError, Object& convertedObject)
   在 System.Web.Script.Serialization.JavaScriptSerializer.Deserialize(JavaScriptSerializer serializer, String input, Type type, Int32 depthLimit)
   在 System.Web.Script.Serialization.JavaScriptSerializer.Deserialize[T](String input)
   在 SyntacticSugar.TypeParseExtenions.JsonToModel[TEntity](String json)
   在 DaZhongTransitionLiquidation.Common.ShanghaiBankAPI.GetShangHaiBankTradingFlow() 位置 E:\Project\DaZhongTransitionLiquidation.Common\ShanghaiBankAPI.cs:行号 26
2019-01-30 09:23:50,307 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:24:13,970 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:24:35,257 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:24:56,551 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:25:17,660 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:25:38,870 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:26:00,161 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:26:21,260 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:26:42,468 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:27:03,751 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:27:24,888 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:27:46,105 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:28:07,440 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:28:28,605 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:28:49,825 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:28:50,609 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:29:10,967 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:29:32,180 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","bf98316b-d64f-4a8c-9788-ce968fcaac3d","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:29:53,337 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:30:14,538 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:30:35,700 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:30:56,856 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:31:18,053 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["a13831b1-1ffd-4d37-bb36-12e4bbbd361f","d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","6d94601d-9c3c-45b7-aa12-ac470bf6dc58","bf98316b-d64f-4a8c-9788-ce968fcaac3d","98ade092-7ee2-4c53-9692-d23b30417fe0","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:31:39,226 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:32:00,380 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:32:21,509 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:32:42,654 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:33:03,798 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:33:24,950 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["a13831b1-1ffd-4d37-bb36-12e4bbbd361f","d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","c1359c24-b50e-4f27-9316-41e9e7474d24","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","cf63d8f5-86c3-4f50-92fa-5f84fa244d07","bd1712ee-6e01-4661-888d-8f941addc125","6d94601d-9c3c-45b7-aa12-ac470bf6dc58","bf98316b-d64f-4a8c-9788-ce968fcaac3d","98ade092-7ee2-4c53-9692-d23b30417fe0","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:33:46,081 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:33:50,855 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:34:07,238 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:34:28,370 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","d9211c60-fd03-4687-97a5-2a7e3ba448e1","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","3d3afc82-7b65-49d2-8d86-665ea86d932b","518a389c-9445-4297-b8f1-66bc933b6807","d271b916-f0ad-43fb-9f46-677638ad5110","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","422dab47-d160-44fd-b2b7-8a7da29f53fd","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","7a355495-b877-450d-ba4a-bf6ef520ef34","ed62db65-e1aa-42b5-bffd-c23e2f811cf4","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","8a3503ef-ca6d-4953-90ab-ef84b323b850","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 09:38:51,110 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:43:51,379 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:48:51,683 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:53:51,964 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 09:58:52,263 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:00:00,932 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"CapitalAccount":"31685803002369318","TradingStartDate":"20190129","TradingEndDate":"20190129"},result:{"success":true,"errmsg":"","data":{"DFJE":8188.11,"JFJE":5430.00,"HUMI":"大众交通（集团）股份有限公司大众出租汽车分公司","DFBS":3,"Detail":[{"CDFG":2,"PATI":"转帐","FSJE":781.00,"JYRQ":"2019年01月29日","FSSJ":"04:17","DFZH":243300133,"DFBZ":"CNY","DFHM":"财付通支付科技有限公司","VCTP":"","VCNO":"","YUER":26023341.21,"YOTU":"财付通转账","BEZH":"","T24F":"BEA190290417058141900"},{"CDFG":1,"PATI":"转账","FSJE":2430.00,"JYRQ":"2019年01月29日","FSSJ":"10:08","DFZH":6222620110013902862,"DFBZ":"CNY","DFHM":"邓瑜","VCTP":"","VCNO":"","YUER":26020911.21,"YOTU":"划款","BEZH":"劳动关系协调员培训费　","T24F":"FT19029162006859"},{"CDFG":1,"PATI":"转账","FSJE":3000.00,"JYRQ":"2019年01月29日","FSSJ":"10:09","DFZH":32403108010059753,"DFBZ":"CNY","DFHM":"上海大众交通汽车修理有限公司","VCTP":"","VCNO":"","YUER":26017911.21,"YOTU":"划款","BEZH":"车辆维修费　","T24F":"FT19029407671269"},{"CDFG":2,"PATI":"转帐","FSJE":6004.76,"JYRQ":"2019年01月29日","FSSJ":"11:19","DFZH":"0000300000000181","DFBZ":"CNY","DFHM":"银联商务股份有限公司客户备付金","VCTP":"","VCNO":"","YUER":26023915.97,"YOTU":"0128-0128费13.24元","BEZH":"","T24F":"BEA190291119205401501"},{"CDFG":2,"PATI":"转帐","FSJE":1402.35,"JYRQ":"2019年01月29日","FSSJ":"11:26","DFZH":"0000300000000181","DFBZ":"CNY","DFHM":"银联商务股份有限公司客户备付金","VCTP":"","VCNO":"","YUER":26025318.32,"YOTU":"0128-0128费3.09元","BEZH":"","T24F":"BEA190291126321761101"}],"ACNO":31685803002369318,"SUMU":5,"JFBS":2}}
2019-01-30 10:00:22,332 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:00:43,763 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:01:05,018 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:01:26,129 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:01:47,351 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:02:08,636 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:02:29,752 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:02:50,960 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:03:12,246 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:03:33,367 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:03:52,522 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:03:54,571 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:04:15,860 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:04:36,986 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:04:58,299 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:05:19,448 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:05:40,612 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","bf98316b-d64f-4a8c-9788-ce968fcaac3d","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:06:01,762 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","cf89a749-7ab1-442f-82f0-2172c152c30e","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:06:22,981 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:06:44,170 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:07:05,297 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:07:26,417 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["a13831b1-1ffd-4d37-bb36-12e4bbbd361f","d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","bd1712ee-6e01-4661-888d-8f941addc125","6d94601d-9c3c-45b7-aa12-ac470bf6dc58","bf98316b-d64f-4a8c-9788-ce968fcaac3d","98ade092-7ee2-4c53-9692-d23b30417fe0","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:07:47,546 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:08:08,701 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:08:29,857 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:08:51,032 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-28", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","46388127-24b3-41fb-9c9d-bb7d9708d70f","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:08:52,811 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:09:12,153 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["785eaa88-6b38-44a6-852c-588ef371e3ed","bd1712ee-6e01-4661-888d-8f941addc125","9ebaf459-be69-4c08-bf36-dfd33d69f1b6"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:09:33,306 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":11, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["a13831b1-1ffd-4d37-bb36-12e4bbbd361f","d0935d6e-fb88-462e-9c59-1fdb9accaa3f","dfceded2-f6be-48f0-870a-26dc281c01f9","c1359c24-b50e-4f27-9316-41e9e7474d24","785eaa88-6b38-44a6-852c-588ef371e3ed","68172697-5f6d-42ec-8b7d-5d1354b221a0","cf63d8f5-86c3-4f50-92fa-5f84fa244d07","bd1712ee-6e01-4661-888d-8f941addc125","6d94601d-9c3c-45b7-aa12-ac470bf6dc58","bf98316b-d64f-4a8c-9788-ce968fcaac3d","98ade092-7ee2-4c53-9692-d23b30417fe0","9ebaf459-be69-4c08-bf36-dfd33d69f1b6","ccf9e12f-f00f-4659-9f31-f052fe89a002"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:09:54,438 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-24", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","518a389c-9445-4297-b8f1-66bc933b6807","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:10:15,563 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-29", "ClearingPlatformReconciliations":["6c28559d-4f9c-432e-8101-080b96daf33e","84aab259-df29-4e55-b83f-17c4705df395","4afaf65e-b637-41ad-8f68-4f8f5655175d","518a389c-9445-4297-b8f1-66bc933b6807","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","189726ae-f368-471d-aae4-a1d3f9559c34","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:10:36,709 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{ "ReceiptCategory":12, "ReconciliationsDate":"2019-01-23", "ClearingPlatformReconciliations":["50839685-dca5-4ee3-ae14-04458895eb32","6c28559d-4f9c-432e-8101-080b96daf33e","eb176293-3f71-4fca-abfc-0ba8731ad163","917a8fe6-5d74-4508-aa51-107f9e886d4d","ec80135f-29f6-4e38-b84f-11dae9c6d8b9","47bd886b-c2a6-4c90-81f2-16ec46295762","84aab259-df29-4e55-b83f-17c4705df395","6dd40dfc-f29e-4f1c-b939-17f3c8081c54","cf89a749-7ab1-442f-82f0-2172c152c30e","d9211c60-fd03-4687-97a5-2a7e3ba448e1","9946eacc-728e-490a-b649-309e12b58e9b","aae2599e-594a-4366-a093-348625f5f34a","e3124499-6162-4092-b185-36ad0eda4898","c35c5257-0106-4fdc-99ed-4c2b4c7e55d3","4afaf65e-b637-41ad-8f68-4f8f5655175d","24caeda5-fb7d-4b07-9997-577292e60f63","1ce5aadf-e649-4ea8-b80b-6191ac8727f0","d475d1fc-921f-4009-8e1b-630d861254bb","3d3afc82-7b65-49d2-8d86-665ea86d932b","518a389c-9445-4297-b8f1-66bc933b6807","d271b916-f0ad-43fb-9f46-677638ad5110","71d42b1f-9522-44d3-b2f1-6b908459b708","61bb6fde-6e46-46c2-9b59-725efc79877d","54119299-1111-4cf9-a5f7-7862beb68dd5","ba674986-edab-45c8-a6a8-800df53cc66f","17bf34cd-27c6-47e0-a5fc-818b910bb1e4","422dab47-d160-44fd-b2b7-8a7da29f53fd","a21cec2b-7a4f-47dc-97ae-8d1f0db867da","41910348-ba5d-4524-bc3e-9f2e2896b2dd","189726ae-f368-471d-aae4-a1d3f9559c34","f3c5397a-5178-4d56-bc06-aa6a1d7e7a74","64ba128b-0d55-4670-8eb9-ad9c08d7b2ea","c61381c7-0246-4f45-ac76-b65cf1bde722","cb764eff-9f44-47cb-bde8-b9f6a7c0547d","46388127-24b3-41fb-9c9d-bb7d9708d70f","bddb14c5-7112-45a4-b05a-bc5181b5c111","7a355495-b877-450d-ba4a-bf6ef520ef34","ed62db65-e1aa-42b5-bffd-c23e2f811cf4","fa3b307f-017e-48c4-b3de-d0b81dc5720e","0302eaf3-122f-495c-98f5-d43d7a240bd5","fa53664b-0a7e-47bc-8641-e4aa7031ae07","eae14caa-2283-439f-8a42-e6fb49c4954a","8a3503ef-ca6d-4953-90ab-ef84b323b850","7d6b9845-bf39-414f-ac53-f3c9bfb474e9"] },result:System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.173.81:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   在 System.Net.WebClient.UploadString(Uri address, String method, String data)
   在 System.Net.WebClient.UploadString(String address, String data)
   在 DaZhongTransitionLiquidation.Areas.ReportManagement.Controllers.ReconciliationReport.ReconciliationReportPack.GetRevenueSystemAmount(DbBusinessDataService DbBusinessDataService, String BankDate, String revenueDate, String channel_Id) 位置 E:\Project\DaZhongTransitionLiquidation\Areas\ReportManagement\Controllers\ReconciliationReport\ReconciliationReportPack.cs:行号 157
2019-01-30 10:13:53,125 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:18:53,436 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:23:53,687 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:28:53,975 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:33:54,242 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:38:54,508 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:43:54,794 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:48:55,102 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
2019-01-30 10:53:55,383 [9] [ERROR] DaZhongTransitionLiquidation.Common.Log4NetWriter - Data:{"OSNO":"1612420190128094804"},result:{"success":true,"errmsg":"","data":{"serialNo":1612420190128094804,"T24D":"","RECO":"0001","REMG":"待授权","T24S":""}}
